tutorials/introductory/usage.py

```diff
@@ -164,7 +164,7 @@
 ###############################################################################
 # or (pyplot-style)
 
-x = np.linspace(0, 2, 100)
+x = np.linspace(0, 2, 100)  # This is the same data from the previous plot.
```

Comment:
how does this comment help give context?

Reply:
I followed @timhoffm's suggestions and have "Sample data." as the comment here instead.